src/System.IO.Packaging/ref/Configurations.props

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netstandard1.3;
```

Comment:
Yes, adding when I think we'll want to ship that live.  Removing when we'd be ok with harvesting.